Server/src/Server.Api/Controllers/LibrariesController.cs

```diff
@@ -40,4 +40,18 @@ public async Task<ActionResult> Post(Guid id)
         await Mediator.Send(new RemoveLibraryCommand(id));
         return NoContent();
     }
+
+    [HttpPost("books")]
+    public async Task<ActionResult> AddBook(AddBookToLibraryCommand command)
```

Comment:
@DontEatRice @jgulinski @bmikolajj nie jestem pewien czy tak jest ładnie, że jest kontroler Libraries, a libraryId jest w body, nie w URLu

Reply:
no według sztuki to powino być {id}/books

Reply:
Ta właśnie wiem, tylko wtedy bym musiał nowy model tworzyć, żeby z niego przekazać bookId i amount do Commanda. No ale coś tam zrobię